src/webaudio/WebAudioContext.ts

```diff
@@ -77,7 +77,7 @@ class WebAudioContext extends Filterable implements IMediaContext
         this._ctx = ctx;
         // ios11 safari's webkitOfflineAudioContext allows only 44100 Hz sample rate
         this._offlineCtx = new WebAudioContext.OfflineAudioContext(1, 2,
-            (win.OfflineAudioContext) ? ctx.sampleRate : 44100);
+            (win.OfflineAudioContext && (ctx.sampleRate >= 8000 && ctx.sampleRate <= 96000)) ? ctx.sampleRate : 44100);
```

Comment:
Does it make more sense to clamp here instead?
```suggestion
            (win.OfflineAudioContext) ? Math.max(8000, Math.min(96000, ctx.sampleRate)) : 44100);
```

Reply:
@bigtimebuddy I have little knowledge about sound sample rate. If you think clamping is more proper, maybe that'll be right:)